packages/forms/src/Components/Component.php

```diff
@@ -38,6 +38,7 @@ protected function getDefaultEvaluationParameters(): array
             'get' => $this->getGetCallback(),
             'livewire' => $this->getLivewire(),
             'model' => $this->getModel(),
+            'old' => $this->getOld(),
```

Comment:
Instead of passing this in all the time, we can just pass it into the `$this->evaluate()` found in the `callAfterStateUpdated()` method

Reply:
Sorry, I'm not really sure about this one.
So instead of passing the old state, do we pass a closure that returns the old state?